ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -351,8 +351,9 @@ impl<'a> Assemble<'a> {
                 // Push [GuardIdx]'s of previous guard failures.
                 if let Some(ref guards) = prevguards {
                     for gidx in guards.iter().rev() {
+                        let g = i64::try_from(<GuardIdx as Into<usize>>::into(*gidx)).unwrap();
```

Comment:
Is this `i64::try_from(usize::from(gidx)).unwrap()` in disguise? If so, please force push an update.

Reply:
It was. Don't ask me how this happened. :laughing: